pay-api/src/pay_api/services/payment_account.py

```diff
@@ -831,29 +831,21 @@ def create_account_event_payload(self, event_type: str, receipt_info: dict = Non
                                      include_pay_info: bool = False):
         """Return event payload for account."""
         payload: Dict[str, any] = {
-            'specversion': '1.x-wip',
-            'type': event_type,
-            'source': f'https://api.pay.bcregistry.gov.bc.ca/v1/accounts/{self.auth_account_id}',
-            'id': f'{self.auth_account_id}',
-            'time': f'{datetime.now()}',
-            'datacontenttype': 'application/json',
-            'data': {
-                'accountId': self.auth_account_id,
-                'accountName': self.name
-            }
+            'accountId': self.auth_account_id,
```

Comment:
`QueueMessageTypes.PAD_ACCOUNT_CREATE.value`
`QueueMessageTypes.NSF_UNLOCK_ACCOUNT.value`
`EFT_AVAILABLE_NOTIFICATION`

need to be tested

Reply:
PAD_ACCOUNT_CREATE works

Reply:
EFT_AVAILABLE works